src/db/connection.cr

```diff
@@ -52,6 +56,9 @@ module DB
       create_transaction
     end
 
+    protected def check
```

Comment:
Could you add some documentation to this method, please? Just a short description what implementing drivers should do here.

Reply:
Done!